bitbots_localization/src/RobotState.cpp

```diff
@@ -8,17 +8,16 @@ RobotState::RobotState() :
     m_XPos(0.0),
     m_YPos(0.0),
     m_SinTheta(0.0),
-    m_CosTheta(1.0),
-    is_explorer_(false) {
+    m_CosTheta(1.0) {
```

Comment:
Why did you remove `is_explorer_(false)`?

Reply:
By accident, thanks for spotting it.